app/api/entities/sample_entity.rb

```diff
@@ -50,7 +50,12 @@ class SampleEntity < ApplicationEntity
       unless: ->(instance, options) { displayed_in_list? }
     )
 
-    expose_timestamps
+    expose :code_log, using: Entities::CodeLogEntity
```

Comment:
Why did you remove the timestamps and instead badly duplicate the exposition of CodeLog?